ar_commands.rb

```diff
@@ -5,27 +5,29 @@
 require_relative "models/painting"
 
 # 1. Create a new Artist (Vincent Van Gogh) and save him to your database
-
+Artist.new(name: "Vincent van Gogh")
 # 2. Create a new Painting (Starry Night) and save it to your database
-
```

Comment:
remember `.new` does not save to the database, would recommend using `.create` here